spec/scopes-and-launch-context/index.md

```diff
@@ -7,9 +7,14 @@ SMART on FHIR's authorization scheme uses OAuth scopes to communicate (and
 negotiate) access requirements. Providing apps with access to broad data sets is consistent with current common practices (e.g. interface engines also provide access to broad data sets); access is also limited based on the privileges of the user in context.  In general, we use scopes for three kinds of data:
 
 1. Clinical data
-2. Contextual data
```

Comment:
@Healthedata1 Why did you remove this list item? Scopes are still the way that an app requests contextual data (e.g. for the Standalone Launch flow).